packages/autocannon-ui-frontend/src/components/ResultSet.jsx

```diff
@@ -51,9 +51,10 @@ const useStyles = makeStyles(theme => ({
 
 export default function ResultSet({ data }) {
   const classes = useStyles()
-
+  
   return (
     <React.Fragment>
+      {data != undefined ?
```

Comment:
why this? in any case, prefer a less cryptic:

```suggestion
      {data &&
```

and remove the empty div at line 253. it's more react idiomatic